docs/src/examples.md

```diff
@@ -112,7 +112,7 @@ y = rand(rng, td)
 Want to fit the flow?
 
 ```@repl normalizing-flows
-using Zygote
+using ForwardDiff
```

Comment:
This seems to be failing at docs build time, see `https://turinglang.org/Bijectors.jl/previews/PR393/examples/`. Probably just needs ForwardDiff to be added as a docs dependency in `docs/Project.toml`, or something like that.